.gitignore

```diff
@@ -1,3 +1,6 @@
 System/HIDAPI.hs
 dist/
 cabal-dev/
+.stack-work/
+*~
```

Comment:
Fellow vi haskeller?  It's rare to find another one in the wild!
This might be better suited to your global gitignore, though :)

Reply:
I must apologize, I copied this over from another project without giving it much thought. I am not really a vi user :)